GraphQL.Server.sln

```diff
@@ -18,7 +18,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{0FD61587
 		nuget.config = nuget.config
 	EndProjectSection
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{382C5C04-A34D-4C81-83D7-584C85FB9356}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".Solution Items", ".Solution Items", "{382C5C04-A34D-4C81-83D7-584C85FB9356}"
```

Comment:
Dot?

Reply:
Yea iI did it so that it gets ordered at the top in the Sln Explr, otherwise it's awkwardly in the middle of other folders.

Reply:
ok

Reply:
ok